tools/RepositoryManagement/DevHome.RepositoryManagement/Models/Commit.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace DevHome.RepositoryManagement.Models;
+
+public class Commit
+{
+    public static readonly Commit DefaultCommit = new(string.Empty, TimeSpan.FromMinutes(0), string.Empty);
+
+    public string Author { get; }
+
+    public TimeSpan TimeSinceCommit { get; }
```

Comment:
Why store `TimeSinceCommit`? This will immediately become out-of-date. Instead, store the actual time of the commit, and you can always calculate the time since then by comparing to `Now`.

Reply:
Good catch.  Changed.